MeatcubeParkingSystem/src/main/java/com/metacube/parking/service/MeatcubeParkingSystemTest.java

```diff
@@ -0,0 +1,68 @@
+package com.metacube.parking.service;
+
+import static org.junit.Assert.*;
+
+import java.sql.SQLIntegrityConstraintViolationException;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.metacube.parking.model.Employee;
+import com.metacube.parking.model.Plan;
+import com.metacube.parking.model.Vehicle;
+
+/**
+ * The Class MeatcubeParkingSystemTest.
+ */
+public class MeatcubeParkingSystemTest {
+
+private Employee employee = new Employee("/images/download.jpg","Aman","Male","amangautam780@gmail.com","1234","1234","8318963298","metacube");
```

Comment:
Can use beforeTest on a method that is called before every test and can then generate dummy data in that method.